hal/include/HAL/SPI.h

```diff
@@ -21,18 +21,18 @@ int32_t HAL_WriteSPI(int32_t port, uint8_t* dataToSend, int32_t sendSize);
 int32_t HAL_ReadSPI(int32_t port, uint8_t* buffer, int32_t count);
 void HAL_CloseSPI(int32_t port);
 void HAL_SetSPISpeed(int32_t port, int32_t speed);
-void HAL_SetSPIOpts(int32_t port, HAL_Bool msb_first,
-                    HAL_Bool sample_on_trailing, HAL_Bool clk_idle_high);
+void HAL_SetSPIOpts(int32_t port, HAL_Bool msbFirst, HAL_Bool sampleOnTrailing,
+                    HAL_Bool clkIdleHigh);
 void HAL_SetSPIChipSelectActiveHigh(int32_t port, int32_t* status);
 void HAL_SetSPIChipSelectActiveLow(int32_t port, int32_t* status);
 int32_t HAL_GetSPIHandle(int32_t port);
 void HAL_SetSPIHandle(int32_t port, int32_t handle);
 
 void HAL_InitSPIAccumulator(int32_t port, int32_t period, int32_t cmd,
```

Comment:
The implementation already has the correct naming convention. Compilers don't care about the name used in the prototype. They could even be omitted and it would still compile.